docs/advanced_usage/advanced_tests.rst

```diff
@@ -15,44 +20,63 @@ an additional reference can be added into the test_run decorator:
+- tests that define all provided tag names with at least one matching tag value for each of the tag names.
+
+The only limitation is to always specify you tags as options in the CLI, i.e. as pairs of tag name
+and tag value. Multiple tag values for a single tag name simply need to be comma-separated (without whitespace).
+
+.. note:: The cli will only allow you to use the character ``-`` instead of ``_`` for tags. If you call for example
     ``--branch-level`` in the cli, you can use following corresponding tags in your test case: ``branch-level``,
     ``branch_level`` or ``branchlevel``.
 
+Find below an example of such a CLI invocation:
+
 .. code:: bash
 
     pykiso -c configuration_file --variant var1,var2 --branch-level daily,nightly
 
-.. table:: Exectuion table for test case tags and cli tag arguments
+
+.. table:: Execution table for test case tags and cli tag arguments
    :widths: auto
 
-   =======================================================  ===============================  ========
-   test case tags                                           cli tags                         executed
-   =======================================================  ===============================  ========
-   "branch_level": ["daily","nightly"]                      branch-level nightly             🗸
-   "branch-level": ["daily","nightly"]                      branch-level nightly             🗸
-   "branch_level": ["daily","nightly"]                      branch-level nightly,daily       🗸
-   "branch_level": ["daily","nightly"]                      branch-level master              ✗
-   "branch_level": ["daily","nightly"],"variant":["var1"]   branch-level daily variant var9  🗸
-   "branch_level": ["daily","nightly"],"variant":["var1"]   branch-level nightly             🗸
-   "branch_level": ["daily","nightly"],"variant":["var1"]   variant var1                     🗸
-   "branch_level": ["daily","nightly"],"variant":["var1"]   variant var2                     ✗
-   "branch_level": ["daily","nightly"],"variant":["var1"]   branch-level daily variant var1  ✗
-   "variant":["var1"]                                       branch-level daily               ✗
-   =======================================================  ===============================  ========
+   ========================================  =============================================================  ========
+   CLI Tags                                  Test case Tags                                                 Executed
+   ========================================  =============================================================  ========
+   none                                      any                                                            🗸
+   ``--branch-level nightly``                ``"branch-level": ["daily", "nightly"]``                       🗸
+   ``--branch-level nightly,daily``          ``"branch_level": ["daily", "nightly"]``                       🗸
```

Comment:
For completeness add this line:
``--branch-level nightly,daily``          ``"branch_level": ["daily"]``                       🗸
Otherwise it is unclear if both tag values need to be specified on a test case to execute.

Reply:
Right! Added it